packages/form-js-viewer/src/features/viewerCommands/behaviors/index.js

```diff
@@ -0,0 +1,9 @@
+// import SomethingBehavior from 'SomethingBehavior';
+
+export default {
+
+  // __init__: [ 'somethingBehavior' ]
+  // somethingBehavior: [ 'type', SomethingBehavior ]
+};
+
+export { /* SomethingBehavior */ };
```

Comment:
This can be removed I guess?

Reply:
Yeah I got rid of it, we can bring in behaviors some time later it's not like we lack the examples for this.